pkg/utils/labels_annotations.go

```diff
@@ -112,6 +121,21 @@ const (
 
 	// LivenessPingerAnnotationName is the name of the pinger configuration
 	LivenessPingerAnnotationName = AlphaMetadataNamespace + "/livenessPinger"
+
+	// ManagedByLabelName is the name of the label applied to all managed objects
+	ManagedByLabelName = "app.kubernetes.io/managed-by"
+
+	// NameLabelName is the name of the label used to indicate the name of the application
+	AppLabelName = "app.kubernetes.io/name"
```

Comment:
You have a discrepancy between the name and the comment. Perhaps half-decided?
And, maybe AppNameLabelName?